header/store/heightsub.go

```diff
@@ -79,7 +80,7 @@ func (hs *heightSub) Pub(headers ...*header.ExtendedHeader) {
 	height := hs.Height()
 	from, to := uint64(headers[0].Height), uint64(headers[ln-1].Height)
 	if height+1 != from {
-		log.Fatal("PLEASE FILE A BUG REPORT: headers given to the heightSub are in the wrong order")
```

Comment:
[for reviewers] I encountered an error after running `make fmt` where `log` was declared twice in this package because [this line](https://github.com/rootulp/celestia-node/blob/100bfe33231a4574878a4b0b4244f0adf978aa08/header/store/store.go#L19) so I added an import alias here.

Reply:
Weird. It wouldn't compile before then

Reply:
On second pass I see what happened: `make fmt` is automatically adding an import for `"log"` but it shouldn't be imported and instead [this line](https://github.com/rootulp/celestia-node/blob/100bfe33231a4574878a4b0b4244f0adf978aa08/header/store/store.go#L19) should be used.

Reply:
Resolved in https://github.com/celestiaorg/celestia-node/pull/1147/commits/e321b855f99df64941e6124fa9b88fa2884e2e63